tests/phpunit/tests/theme/wpThemeJsonResolver.php

```diff
@@ -365,12 +369,56 @@ function test_get_user_data_from_wp_global_styles_does_not_use_uncached_queries(
 		$this->assertNotEmpty( $user_cpt );
 
 		$query_count = count( $this->queries );
-		for ( $i = 0; $i < 3; $i++ ) {
-			WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles( wp_get_theme() );
+		for ( $i = 0; $i < 3; $i ++ ) {
```

Comment:
Should remove the space

```suggestion
		for ( $i = 0; $i < 3; $i++ ) {
```